optax/_src/alias.py

```diff
@@ -1082,12 +1082,73 @@ def rprop(
     The corresponding `GradientTransformation`.
   """
   return combine.chain(
-      transform.scale_by_rprop(
-          learning_rate=learning_rate,
-          eta_minus=eta_minus,
-          eta_plus=eta_plus,
-          min_step_size=min_step_size,
-          max_step_size=max_step_size,
-      ),
-      transform.scale(-1.0),
+    transform.scale_by_rprop(
+      learning_rate=learning_rate,
+      eta_minus=eta_minus,
+      eta_plus=eta_plus,
+      min_step_size=min_step_size,
+      max_step_size=max_step_size,
+    ),
+    transform.scale(-1.0),
+  )
+
+
+def ftrl(
+  learning_rate: float = 1e-3,
+  initial_accumulator_value: float = 0,
+  lambda_1: float = 0,
+  lambda_2: float = 0,
+  beta: float = 1
+  ) -> base.GradientTransformation:
+  # pylint: disable=line-too-long
+  r"""The FTRL optimizer.
+  
+  FTRL, or Follow the Regularized Leader, is an optimization algorithm developed
+  at Google for click-through rate prediction in the early 2010s. It is most
+  suitable for shallow models with large and sparse feature spaces.
+  
+  McMahan et al. gives a closed form update for the parameters as,
+  
+  .. math::
+  
+    \begin{align*}
```

Comment:
use & to align equations:

```
    \begin{align*}
      w_{t,i} &= \begin{cases} 
        0 & \text{if } |z_i| \leq \lambda_1 \\
        -\left( \frac{\beta + \sqrt{n_i}}{\alpha} + \lambda_2 \right)^{-1} 
          \left( z_i - \text{sgn}(z_i)\lambda_1 \right) & \text{otherwise}.
      \end{cases} \\
      \sigma_i &= \frac{1}{\alpha} \left( \sqrt{n_i + g_i^2} - \sqrt{n_i} \right) \\
      z_i &\leftarrow z_i + g_i - \sigma_i w_{t,i} \\
      n_i &\leftarrow n_i + g_i^2
    \end{align*}
```

 this way it looks like 
<img width="506" alt="image" src="https://github.com/google-deepmind/optax/assets/277639/77455415-9a1b-486e-80bf-553e724bd2a4">
instead of 
<img width="508" alt="image" src="https://github.com/google-deepmind/optax/assets/277639/d2cf9ef9-393e-421a-97d4-e89459aa7f18">